setup.py

```diff
@@ -30,7 +30,7 @@
       author_email='mongodb-user@googlegroups.com',
       url='https://github.com/mongodb-labs/elastic-doc-manager',
       packages=["mongo_connector", "mongo_connector.doc_managers"],
-      install_requires=['mongo-connector >= 2.3.0', 'elasticsearch >= 1.2, < 2.0.0'],
+      install_requires=['mongo-connector >= 2.3.0', 'elasticsearch >= 1.2, < 2.0.0', 'requests >= 2.5.1', 'requests-aws4auth >= 0.7'],
```

Comment:
This might be better in an `extras_require`, so that aws4auth can be installed alongside elastic-doc-manager selectively: http://setuptools.readthedocs.io/en/latest/setuptools.html#declaring-extras-optional-features-with-their-own-dependencies